drivers/pci/pci.c

```diff
@@ -157,6 +159,46 @@ void rtc_set_checksum(int range_start, int range_end, int cks_loc)
 	CMOS_WRITE(((sum >> 0) & 0x0ff), cks_loc+1);
 }
 
+/* SMSC LPC47M157 Super I/O functions */
+/* pin and register compatible with LPC47M192 */
+void LpcEnableConfiguration(int Enable)
+{
+	IoOutputByte(0x2e, Enable ? 0x55 : 0xAA);
+}
+
+int LpcGetSerialState()
+{
+	// Select serial device
+	IoOutputByte(0x2e, 0x07);
+	IoOutputByte(0x2f, 0x04);
```

Comment:
nit: Consider `LpcWriteConfiguration` and `LpcReadConfiguration` to abstract these repeating 0x2E/0x2F writes (not sure if that makes any sense - I hardly know anything about SuperIO)